contrib/asan.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Helper script to build and run neovim with Address Sanitizer enabled.
+# You may read more information in src/nvim/README.md in the section "Build
+# with ASAN".
+
+shopt -s nullglob
+
+root_path=$(git rev-parse --show-toplevel)
+log_path=$(mktemp -d)
+export CC='clang'
+
+# Change to detect_leaks=1 to detect memory leaks (slower).
+export ASAN_OPTIONS="detect_leaks=0:log_path=$log_path/asan"
+
+# Show backtraces in the logs.
+export UBSAN_OPTIONS="print_stacktrace=1"
+
+make -C "$root_path" CMAKE_INSTALL_PREFIX="$root_path" CMAKE_EXTRA_FLAGS="-DCLANG_ASAN_UBSAN=ON"
+make -C "$root_path" install
```

Comment:
I think this line is not needed, since it would need sudo permissions to install runtime files in `/usr/`.

Reply:
(and no need to set `CMAKE_INSTALL_PREFIX` then)